collect_app/src/main/java/org/odk/collect/android/external/InstanceProvider.java

```diff
@@ -132,6 +133,10 @@ public Uri insert(@NonNull Uri uri, ContentValues initialValues) {
             throw new IllegalArgumentException("Unknown URI " + uri);
         }
 
+        if (initialValues.containsKey(DatabaseInstanceColumns.SUBMISSION_URI)) {
+            throw new SecurityException();
```

Comment:
I'm wondering if it would be helpful to log this event to understand if people do that and how often.

Reply:
We'll get crash logs so I don't think we need to log anything else.